### PR TITLE
Add solution comments

### PR DESCRIPTION
**Pull request by iHiD**
@kntsoriano https://www.loom.com/share/c5228e31127a4e4db0df6a14f925b5f6

### CODE REVIEW

File: app/views/tracks/community_solutions/show.html.haml [hunk 1]
```diff
@@ -71,6 +71,40 @@
...
+        - @comments.each do |comment|
+          .comment
+            %header.flex.items-center.mb-16
+              = avatar(comment.author)
+              .flex.flex-col
+                .flex.items-center
+                  .text-h6.mr-8= comment.author.handle
+                  = render ViewComponents::Reputation.new(comment.author.formatted_reputation, size: 'small')
+                .text-textColor6.leading-160 #{time_ago_in_words(comment.created_at)} ago
+            .c-textual-content.--small= raw comment.content_html
```
Comment (iHiD) [inline app/views/tracks/community_solutions/show.html.haml]: @kntsoriano This is the bit to turn into React.

**Comment by kntsoriano** [comment]
@iHiD Should be ready. Do you want the following to be added?

1. Websockets
2. Scrolling to post when submitted.
3. New messages button just like in the mentor discussion

**Comment by iHiD** [comment]
@kntsoriano Thanks!

>  Do you want the following to be added?

Not for now. Could you open an issue for us to do this in the future pls?

**Comment by iHiD** [comment]
@kntsoriano Could you get the JS tests green pls. (I forced pushed to rebase btw)

**Pull request merged**